packages/react/src/themes/tokens.ts

```diff
@@ -1,11 +1,182 @@
-import { main } from './main';
 import { Theme } from './theme';
 
+/** TODO - adjust components to use correct color names as per below */
+const greys = {
+    white: '#FFFFFF',
+    'neutral-02': '#FAFAFA',
+    'neutral-05': '#F1F2F2',
+    'neutral-15': '#DBDEE1',
+    'neutral-30': '#B7BBC2',
+    'neutral-50': '#878F9A',
+    'neutral-65': '#60666E',
+    'neutral-90': '#1B1C1E',
+    black: '#000000',
+};
+
+const main = {
+    'brand-05': '#E0F0F9',
+    'brand-20': '#84C6EA',
+    'brand-50': '#006296',
+    'brand-70': '#003A5A',
+    'brand-80': '#012639',
+    'accent-20': '#F9B6B2',
+    'accent-50': '#EF483E',
+    'accent-70': '#D41F14',
+};
+
+const notifications = {
+    'success-05': '#F6FCF8',
+    'success-20': '#8ADDA9',
+    'success-50': '#008533',
+    'success-70': '#004F1E',
+    'alert-05': '#FDF7F6',
+    'alert-20': '#F99D99',
+    'alert-50': '#CD2C23',
+    'alert-70': '#7B1A15',
+    'warning-05': '#FFFBF5',
+    'warning-20': '#FFDD99',
+    'warning-50': '#F5A200',
+    'warning-70': '#9E6900',
+    'informative-05': '#F9F7FB',
+    'informative-20': '#CFC1E3',
+    'informative-50': '#602FA0',
+    'informative-70': '#3A1C60',
+};
+
 export const tokens: Theme['tokens'] = {
-    'focus-box-shadow': `0 0 0 2px ${main['primary-1.2']}`,
-    'focus-border-box-shadow': ` 0 0 0 1px ${main['primary-1.1']}, 0 0 0 3px ${main['primary-1.2']}`,
-    'focus-border-box-shadow-inset': `inset 0 0 0 2px ${main['primary-1.2']}, inset 0 0 0 3px ${main['primary-1.1']}`,
-    'focus-border': `${main['primary-1.1']}`,
+    greys,
+    main,
+    notifications,
+    'focus-box-shadow': `0 0 0 2px ${main['brand-20']}`,
+    'focus-border-box-shadow': ` 0 0 0 1px ${main['brand-50']}, 0 0 0 3px ${main['brand-20']}`,
+    'focus-border-box-shadow-inset': `inset 0 0 0 2px ${main['brand-20']}, inset 0 0 0 3px ${main['brand-50']}`,
+    'focus-border': `${main['brand-50']}`,
     'modal-overlay-background-color': 'rgba(0, 0, 0, 0.75)',
     'overlay-box-shadow': '0 10px 20px 0 rgba(0, 0, 0, 0.19)',
+    'bt-primary': {
```

Comment:
I believe it's better to avoid abbreviation and use `button-primary` instead. 

I think it could become pretty messy if we also use abbr. on other components (ex: tt, dp, pg). Otherwise, if we don't plan to use abbr. on all others components, in that case button should not be an exception, just find a naming we could be consistent.

Reply:
done